tests/subsys/settings/functional/particle_xenon.overlay

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019 Peter Bigot Consulting, LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/delete-node/ &storage_partition;
+
+&mx25l32 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		littlefs_partition: partition@0 {
```

Comment:
We don't natively support littlefs in Zephyr. Perhaps it would be better to either make this neutral or else add support (via west) for this filesystem? We know of serious shortcomings with NFFS that need to be addressed, so adding support for littlefs would be great.

Reply:
I'm already a fair way along to littlefs, based on @jimparis's work, but in this case the Particle devices come with a littlefs v1 partition in that location.  So it's correct as is, whether or not we add it to Zephyr.